public class EntregadroApp implements Observer {
    @Override 
    public void update(Pedido pedido) {
        System.out.println(
            "Entregador recebeu atualização: Pedido está " + pedido.getStatus()
        );
    }
}
